build/NuSpecs/WindowsAppSDK-Nuget-Native.WinRt.props

```diff
@@ -24,6 +24,11 @@
       <HintPath>$(MSBuildThisFileDirectory)..\..\lib\native\Microsoft.Windows.System.winmd</HintPath>
       <Implementation Condition="'$(WindowsAppSDKFrameworkPackage)' != 'true'">$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(_WindowsAppSDKFoundationPlatform)\native\Microsoft.WindowsAppSDK.dll</Implementation>
       <IsWinMDFile>true</IsWinMDFile>
+    </Reference>
+	<Reference Include="Microsoft.Windows.System.Power.winmd">
+      <HintPath>$(MSBuildThisFileDirectory)..\..\lib\native\Microsoft.Windows.System.Power.winmd</HintPath>
```

Comment:
Can this target be on uap10.0?

I've been working on a change to move these winmds files to uap10.0
https://github.com/microsoft/WindowsAppSDK/pull/1219

Reply:
Done